utils/compareToNumber.js

```diff
@@ -0,0 +1,23 @@
+import { RULES } from "../constants/messages.js";
+const compareToNumber = (comNum, userNum) => {
```

Comment:
```suggestion
const getBaseballCount = (comNum, userNum) => {
```
`compareToNumber`라는 함수 이름만 봤을 때 2개의 숫자를 받아서 동일한지 비교하는 함수일 것 같은데 실제로는 스트라이크, 볼의 카운트를 세기 때문에 함수 네이밍을 수정해봐도 좋을 것 같아요!